data/helpers.d/utils

```diff
@@ -67,28 +74,34 @@ ynh_backup_before_upgrade () {
 	backup_number=1
 	local old_backup_number=2
 	local app_bck=${app//_/-}	# Replace all '_' by '-'
+	NO_BACKUP_UPGRADE=${NO_BACKUP_UPGRADE:-0}
 
-	# Check if a backup already exists with the prefix 1
-	if sudo yunohost backup list | grep -q $app_bck-pre-upgrade1
-	then
-		# Prefix becomes 2 to preserve the previous backup
-		backup_number=2
-		old_backup_number=1
-	fi
+        if [ "$NO_BACKUP_UPGRADE" -eq 0 ]
+        then
+		# Check if a backup already exists with the prefix 1
+		if sudo yunohost backup list | grep -q $app_bck-pre-upgrade1
+		then
+			# Prefix becomes 2 to preserve the previous backup
+			backup_number=2
+			old_backup_number=1
+		fi
 
-	# Create backup
-	sudo yunohost backup create --ignore-system --apps $app --name $app_bck-pre-upgrade$backup_number --verbose
-	if [ "$?" -eq 0 ]
-	then
-		# If the backup succeeded, remove the previous backup
-		if sudo yunohost backup list | grep -q $app_bck-pre-upgrade$old_backup_number
+		# Create backup
+		sudo yunohost backup create --ignore-system --apps $app --name $app_bck-pre-upgrade$backup_number --verbose
+		if [ "$?" -eq 0 ]
 		then
-			# Remove the previous backup only if it exists
-			sudo yunohost backup delete $app_bck-pre-upgrade$old_backup_number > /dev/null
+			# If the backup succeeded, remove the previous backup
+			if sudo yunohost backup list | grep -q $app_bck-pre-upgrade$old_backup_number
+			then
+				# Remove the previous backup only if it exists
+				sudo yunohost backup delete $app_bck-pre-upgrade$old_backup_number > /dev/null
+			fi
+		else
+			ynh_die "Backup failed, the upgrade process was aborted."
 		fi
-	else
-		ynh_die "Backup failed, the upgrade process was aborted."
-	fi
+        else
+                echo "\$NO_BACKUP_UPGRADE is set, backup will be avoided. Be careful, this upgrade is going to be operated without securi$
```

Comment:
the end of this line seems buggy

Reply:
Looks like a bad copy paste...
I'm fixing that